tests/coupled/static/smith_g_2deg/generate_smith_g_2deg.py

```diff
@@ -208,7 +196,7 @@ def generate_aero_file():
 
     working_elem = 0
     working_node = 0
-    # right wing (surface 0, beam 0)
+
```

Comment:
my take is to leave these in - it helps with understanding the model generation procedure which if now there's no simplification coming up the pipeline seems to have a big learning curve.